<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ChesSter-ПК" name="Command line test" tests="23" failures="2" timestamp="15 Jun 2016 07:35:20 GMT" time="6.319" errors="0">
  <testcase name="@BeforeMethod setUp" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <failure type="org.dbunit.dataset.DataSetException" message="java.net.MalformedURLException">
      <![CDATA[org.dbunit.dataset.DataSetException: java.net.MalformedURLException
	at org.dbunit.dataset.xml.FlatXmlProducer.produce(FlatXmlProducer.java:169)
	at org.dbunit.dataset.CachedDataSet.<init>(CachedDataSet.java:71)
	at org.dbunit.dataset.xml.FlatXmlDataSet.<init>(FlatXmlDataSet.java:200)
	at org.dbunit.dataset.xml.FlatXmlDataSet.<init>(FlatXmlDataSet.java:187)
	at com.websystique.springmvc.dao.EmployeeDaoImplTest.getDataSet(EmployeeDaoImplTest.java:22)
	at com.websystique.springmvc.dao.EntityDaoImplTest.setUp(EntityDaoImplTest.java:27)
java.net.MalformedURLException
	at java.net.URL.<init>(URL.java:619)
	at java.net.URL.<init>(URL.java:482)
	at java.net.URL.<init>(URL.java:431)
	at org.apache.xerces.impl.XMLEntityManager.setupCurrentEntity(Unknown Source)
	at org.apache.xerces.impl.XMLVersionDetector.determineDocVersion(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source)
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source)
	at org.dbunit.dataset.xml.FlatXmlProducer.produce(FlatXmlProducer.java:145)
	at org.dbunit.dataset.CachedDataSet.<init>(CachedDataSet.java:71)
	at org.dbunit.dataset.xml.FlatXmlDataSet.<init>(FlatXmlDataSet.java:200)
	at org.dbunit.dataset.xml.FlatXmlDataSet.<init>(FlatXmlDataSet.java:187)
	at com.websystique.springmvc.dao.EmployeeDaoImplTest.getDataSet(EmployeeDaoImplTest.java:22)
	at com.websystique.springmvc.dao.EntityDaoImplTest.setUp(EntityDaoImplTest.java:27)
Caused by: java.lang.NullPointerException
	at java.net.URL.<init>(URL.java:524)
	... 50 more
... Removed 72 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod springTestContextBeforeTestMethod" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod springTestContextBeforeTestMethod -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod springTestContextBeforeTestMethod" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod springTestContextBeforeTestMethod -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod springTestContextBeforeTestMethod" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod springTestContextBeforeTestMethod -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod springTestContextBeforeTestMethod" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod springTestContextBeforeTestMethod -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod springTestContextBeforeTestMethod" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod springTestContextBeforeTestMethod -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="deleteEmployee" time="0.156" classname="com.websystique.springmvc.controller.AppControllerTest"/>
  <testcase name="editEmployee" time="0.051" classname="com.websystique.springmvc.controller.AppControllerTest"/>
  <testcase name="listEmployees" time="0.008" classname="com.websystique.springmvc.controller.AppControllerTest">
    <failure type="java.lang.AssertionError" message="expected [[Employee [id=1, name=Temirlan, phone=554362, city=Astana, address=Almaty 17], Employee [id=2, name=Duman, phone=995541, city=Astana, address=null]]] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [[Employee [id=1, name=Temirlan, phone=554362, city=Astana, address=Almaty 17], Employee [id=2, name=Duman, phone=995541, city=Astana, address=null]]] but found [null]
	at com.websystique.springmvc.controller.AppControllerTest.listEmployees(AppControllerTest.java:62)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- listEmployees -->
  <testcase name="newEmployee" time="0.001" classname="com.websystique.springmvc.controller.AppControllerTest"/>
  <testcase name="saveEmployeeWithSuccess" time="0.001" classname="com.websystique.springmvc.controller.AppControllerTest">
    <failure type="java.lang.AssertionError" message="expected [Employee Axel registered successfully] but found [Employee Temirlan registered successfully]">
      <![CDATA[java.lang.AssertionError: expected [Employee Axel registered successfully] but found [Employee Temirlan registered successfully]
	at com.websystique.springmvc.controller.AppControllerTest.saveEmployeeWithSuccess(AppControllerTest.java:96)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- saveEmployeeWithSuccess -->
  <testcase name="saveEmployeeWithValidationError" time="0.0" classname="com.websystique.springmvc.controller.AppControllerTest"/>
  <testcase name="saveEmployeeWithValidationErrorNonUniqueSSN" time="0.003" classname="com.websystique.springmvc.controller.AppControllerTest"/>
  <testcase name="updateEmployeeWithSuccess" time="0.002" classname="com.websystique.springmvc.controller.AppControllerTest"/>
  <testcase name="updateEmployeeWithValidationError" time="0.001" classname="com.websystique.springmvc.controller.AppControllerTest"/>
  <testcase name="updateEmployeeWithValidationErrorNonUniqueSSN" time="0.0" classname="com.websystique.springmvc.controller.AppControllerTest"/>
  <testcase name="deleteEmployeeByInvalidSsn" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- deleteEmployeeByInvalidSsn -->
  <testcase name="deleteEmployeeBySsn" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- deleteEmployeeBySsn -->
  <testcase name="findAllEmployees" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- findAllEmployees -->
  <testcase name="findById" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- findById -->
  <testcase name="findEmployeeBySsn" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- findEmployeeBySsn -->
  <testcase name="saveEmployee" time="0.0" classname="com.websystique.springmvc.dao.EmployeeDaoImplTest">
    <skipped/>
  </testcase> <!-- saveEmployee -->
  <testcase name="deleteEmployeeBySsn" time="0.016" classname="com.websystique.springmvc.service.EmployeeServiceImplTest"/>
  <testcase name="findAllEmployees" time="0.001" classname="com.websystique.springmvc.service.EmployeeServiceImplTest"/>
  <testcase name="findById" time="0.001" classname="com.websystique.springmvc.service.EmployeeServiceImplTest"/>
  <testcase name="findEmployeeBySsn" time="0.001" classname="com.websystique.springmvc.service.EmployeeServiceImplTest"/>
  <testcase name="isEmployeeSsnUnique" time="0.001" classname="com.websystique.springmvc.service.EmployeeServiceImplTest"/>
  <testcase name="saveEmployee" time="0.001" classname="com.websystique.springmvc.service.EmployeeServiceImplTest"/>
  <testcase name="updateEmployee" time="0.001" classname="com.websystique.springmvc.service.EmployeeServiceImplTest"/>
</testsuite> <!-- Command line test -->
